docs/retention.md

```diff
@@ -0,0 +1,26 @@
+# Retention Settings
+
+By default, LME will configure an index lifecycle policy that will delete
+indexes based on estimated disk usage. Initially, 80% of the disk will be used
+for the indices, with an assumption that a day of logs will use 1Gb of disk
+space.
```

Comment:
Worth mentioning must be below 85% [Elasticsearch disk watermarking](https://www.elastic.co/guide/en/elasticsearch/reference/current/disk-allocator.html)